model/flow/ledger.go

```diff
@@ -14,6 +16,13 @@ func (r *RegisterID) String() string {
 	return fmt.Sprintf("%x/%x/%x", r.Owner, r.Controller, r.Key)
 }
 
+// Bytes returns a bytes representation of the RegisterID.
+//
+// the encoding uses the injective fingerprint module.
+func (r *RegisterID) Bytes() []byte {
+	return fingerprint.Fingerprint(r)
```

Comment:
how fast is this ?

Reply:
This is the same RLP function used to encode all structures in all `.ID()` calls. I'm assuming it is fast otherwise ID computation would be slow.